components/bottom-nav/Android/Android-bottom-nav.md

```diff
@@ -1,13 +1,12 @@
 <!--docs:
-title: "Bottom navigation"
+title: "Bottom Navigation"
```

Comment:
Google writing standards uses sentence case for titles: we're sticking to "Bottom navigation" as opposed to "Bottom Navigation."